cmd/eventing-operator/kodata/knative-eventing/1-eventing.yaml

```diff
@@ -2768,7 +2768,7 @@ webhooks:
     service:
       name: eventing-webhook
       namespace: knative-eventing
-  failurePolicy: Fail
+  failurePolicy: Ignore
```

Comment:
In the past we have ONLY dones this for the sinkbinding 


please let's just use sinkbinding, for now

Reply:
Done